utils/mariadb/Makefile

```diff
@@ -0,0 +1,539 @@
+	ha_federatedx \
+	ha_spider \
+	ha_test_sql_discovery \
+	handlersocket \
+	libdaemon_example \
+	locales \
+	metadata_lock_info \
+	mypluglib \
+	qa_auth_client \
+	qa_auth_interface \
+	qa_auth_server \
+	query_cache_info \
+	query_response_time \
+	semisync_master \
+	semisync_slave \
+	server_audit \
+	simple_password_check \
+	sql_errlog \
+	wsrep_info
+
+PKG_CONFIG_DEPENDS := \
+	$(patsubst %,CONFIG_PACKAGE_$(PKG_NAME)-server-plugin-%,$(MARIADB_SERVER_PLUGINS)) \
+	CONFIG_PACKAGE_mariadb-server
+
+plugin-adt_null                 := PLUGIN_AUDIT_NULL
+plugin-auth_0x0100              := PLUGIN_AUTH_0X0100
+plugin-auth_ed25519             := PLUGIN_AUTH_ED25519
+plugin-auth_socket              := PLUGIN_AUTH_SOCKET
+plugin-auth_test_plugin         := PLUGIN_AUTH_TEST_PLUGIN
+plugin-client_ed25519           := PLUGIN_CLIENT_ED25519
+plugin-debug_key_management     := PLUGIN_DEBUG_KEY_MANAGEMENT
+plugin-dialog_examples          := PLUGIN_DIALOG_EXAMPLES
+plugin-disks                    := PLUGIN_DISKS
+plugin-example_key_management   := PLUGIN_EXAMPLE_KEY_MANAGEMENT
+plugin-feedback                 := PLUGIN_FEEDBACK
+plugin-file_key_management      := PLUGIN_FILE_KEY_MANAGEMENT
+plugin-ha_archive               := PLUGIN_ARCHIVE
+plugin-ha_blackhole             := PLUGIN_BLACKHOLE
+plugin-ha_connect               := PLUGIN_CONNECT
+plugin-ha_example               := PLUGIN_EXAMPLE
+plugin-ha_federated             := PLUGIN_FEDERATED
+plugin-ha_federatedx            := PLUGIN_FEDERATEDX
+plugin-ha_spider                := PLUGIN_SPIDER
+plugin-ha_test_sql_discovery    := PLUGIN_TEST_SQL_DISCOVERY
+plugin-handlersocket            := PLUGIN_HANDLERSOCKET
+plugin-libdaemon_example        := PLUGIN_DAEMON_EXAMPLE
+plugin-locales                  := PLUGIN_LOCALES
+plugin-metadata_lock_info       := PLUGIN_METADATA_LOCK_INFO
+plugin-mypluglib                := PLUGIN_FTEXAMPLE
+plugin-qa_auth_client           := PLUGIN_QA_AUTH_CLIENT
+plugin-qa_auth_interface        := PLUGIN_QA_AUTH_INTERFACE
+plugin-qa_auth_server           := PLUGIN_QA_AUTH_SERVER
+plugin-query_cache_info         := PLUGIN_QUERY_CACHE_INFO
+plugin-query_response_time      := PLUGIN_QUERY_RESPONSE_TIME
+plugin-semisync_master          := PLUGIN_SEMISYNC_MASTER
+plugin-semisync_slave           := PLUGIN_SEMISYNC_SLAVE
+plugin-server_audit             := PLUGIN_SERVER_AUDIT
+plugin-simple_password_check    := PLUGIN_SIMPLE_PASSWORD_CHECK
+plugin-sql_errlog               := PLUGIN_SQL_ERRLOG
+plugin-wsrep_info               := PLUGIN_WSREP_INFO
+
+MARIADB_CLIENT := \
+	mysql \
+	mysqlcheck
+
+MARIADB_CLIENT_EXTRA := \
+	mariabackup \
+	mysql_find_rows \
+	mysql_fix_extensions \
+	mysql_waitpid \
+	mysqlaccess \
+	mysqladmin \
+	mysqldump \
+	mysqldumpslow \
+	mysqlimport \
+	mysqlshow \
+	mysqlslap \
+	mytop
+
+MARIADB_SERVER := \
+	innochecksum \
+	mysql_install_db \
+	mysql_upgrade \
+	mysqld
+
+MARIADB_SERVER_EXTRA := \
+	aria* \
+	msql2mysql \
+	my_print_defaults \
```

Comment:
can you move this to basic so mysql_install_db --force won't give fatal error ?

Reply:
I was thinking only for my_print_defaults

Reply:
Got you, thanks!